scripts/wrist_test.py

```diff
@@ -0,0 +1,221 @@
+               'elbow_right_axis0', 'elbow_right_axis1',
+               'wrist_right_axis0', 'wrist_right_axis1', 'wrist_right_axis2']
+
+test_lr = "right"
+notest_lr = "left" if test_lr == "right" else "right"
+
+
+def hard_reset():
+    msg = JointState()
+
+    msg.name = joints_name
+    msg.position = [0]*len(msg.name)
+    msg.velocity = [0]*len(msg.name)
+    msg.effort = [0]*len(msg.name)
+    publisher.publish(msg)
+
+    rospy.loginfo("Hard reset")
+    rospy.sleep(2.0)
+
+
+def soft_reset():
+    pass
+
+
+def hand_shake(lr="right"):
+    msg = JointState()
+    msg.name = ["elbow_"+lr+"_axis0"]
+    msg.velocity = [0]*len(msg.name)
+    msg.effort = [0]*len(msg.name)
+    ranges = [0.6, 1.0]
+    samples = 50
+    for i in range(10):
+        if not i % 2:
+            rospy.loginfo("up")
+            msg.name = [ "elbow_"+lr+"_axis0"]
+            # msg.position = [1.1]
+            # msg.velocity = [0]*len(msg.name)
+            # msg.effort = [0]*len(msg.name)
+
+            el_axis1_targets = np.linspace(ranges[0], ranges[-1], samples)
+
+            for t2 in el_axis1_targets:
+                msg.name = [ "elbow_"+lr+"_axis0"]
+                msg.position = [t2]
+                # print(t2)
+                msg.velocity = [0]
+                msg.effort = [0]
+                publisher.publish(msg)
+                rate.sleep()
+                # rospy.sleep(0.15)
+
+            # rospy.loginfo(msg)
+            # rospy.sleep(1.0)
+            # input()
+        else:
+            rospy.loginfo("down")
+            sh_axis1_targets = np.linspace(-1.0,-1.0, 5)
+            el_axis1_targets = np.linspace(ranges[-1], ranges[0], samples)
+
+            for t2 in el_axis1_targets:
+                msg.name = [ "elbow_"+lr+"_axis0"]
+                msg.position = [t2]
+                # print(t2)
+                msg.velocity = [0]
+                msg.effort = [0]
+                publisher.publish(msg)
+                rate.sleep()
+                # rospy.sleep(0.1)
+            # rospy.sleep(2.0)
+            # msg.position = [-0.3]
+
+
+def move_shoulder_elbow(lr="right", elbow=True):
+    msg = JointState()
+
+    msg.name = ["shoulder_"+lr+"_axis0"]
+    if elbow:
+        msg.name.append("elbow_"+lr+"_axis0")
+
+    targets = np.linspace(0.0, -1.1, 200)
+    targets_el = np.linspace(0, 0.6, 200)
+
+    if elbow:
+        for t, te in zip(targets,targets_el):
+            msg.position = [t, te]
+            msg.velocity = [0]*len(msg.name)
+            msg.effort = [0]*len(msg.name)
+            publisher.publish(msg)
+            rate.sleep()
+    else:
+        for t in targets:
+            msg.position = [t]
+        msg.velocity = [0]*len(msg.name)
+        msg.effort = [0]*len(msg.name)
+        publisher.publish(msg)
+        rate.sleep()
+
+
+def move_wrist(lr="right", axis=0):
+    msg = JointState()
+
+    ranges = [-1.5, -0.05]
+    samples = 200
```

Comment:
this should be a param too

Reply:
Done

Reply:
i dont see `samples` in the list of parameters. where is it done?